src/full/Agda/TypeChecking/Monad/Base.hs

```diff
@@ -1353,6 +1353,8 @@ data ModuleInfo = ModuleInfo
     -- be importable.
   , miMode       :: ModuleCheckMode
     -- ^ The `ModuleCheckMode` used to create the `Interface`
+  , miSourceFile :: SourceFile
+    -- ^ The file belonging to the module.
```

Comment:
We could also have gotten the `SourceFile` from `TopLevelModuleName` via `ModuleToSource`.